src/satchip/chip_sentinel2.py

```diff
@@ -131,6 +117,16 @@ def get_scenes(
     return valid_scenes
 
 
+def get_latest_image_versions(items: list[Item]) -> list[Item]:
+    brief_ids = [item.id[:-5] for item in items]
+    latest_items = []
+    for brief_id in set(brief_ids):
+        matching_items = [item for item in items if item.id.startswith(brief_id)]
+        latest_item = max(matching_items, key=lambda x: int(x.properties['s2:sequence']))
```

Comment:
I would name the key function to be more clear what were getting the max of.
```suggestion
        def get_s2_version(item):
            return item.properties['s2:sequence']
        latest_item = max(matching_items, key=get_s2_version)
```